packages/bzBond-server/test/routes.test.js

```diff
@@ -0,0 +1,54 @@
+import { expect } from "chai";
+import build from "../app.js";
+
+describe("routes", function () {
+  describe("/code", function () {
+    it("evaluates javascript", async function () {
+      const app = await build({ options: { logger: false } });
+
+      const response = await app.inject({
+        method: "POST",
+        url: "/code",
+        payload: {
+          jsCode: "2 + 2",
+        },
+      });
+
+      expect(response.body).to.eq("4");
+    });
+  });
+
+  describe("/function", function () {
+    it("calls the function with no parameters", async function () {
+      const app = await build({ options: { logger: false } });
+
+      const response = await app.inject({
+        method: "POST",
+        url: "/function",
+        payload: {
+          func: "foo",
+        },
+      });
+
+      expect(JSON.parse(response.body).func).to.eq("foo");
+    });
+
+    it("calls the function with no parameters", async function () {
```

Comment:
```suggestion
    it("calls the function with parameters", async function () {
```

Reply:
Should the second one be with parameters? There's an array of args (`["bar", "baz"]`).

Reply:
Good catch! Addressed